dist/.magento.env.yaml

```diff
@@ -635,3 +635,25 @@
 #            variables:                                                                                               #
 #              ENV_ENVIRONMENT: "MAGENTO_CLOUD_ENVIRONMENT"                                                           #
 #######################################################################################################################
+# ERROR_REPORT_DIR_NESTING_LEVEL - The number of subdirectories that will be created to save the report.              #
+#                                                                                                                     #
+# If we have the report name as hash sha256('') = 44ffb1087a44e61b018b3cdee72284d017f22e52755c24e5c85cbac1647ae7a7    #
```

Comment:
I would add link to the documentation about this configuration or if it was not released yen, then copy text which should be there

Reply:
The property ERROR_REPORT_DIR_NESTING_LEVEL is described in the PR https://github.com/magento/devdocs/pull/5485 (https://magento2.atlassian.net/browse/MAGECLOUD-4327).
A page with a detailed description was created "Directories Nesting Level Changes for Error Reporting" and a description about the property ERROR_REPORT_DIR_NESTING_LEVEL was added to the page "Building Variables"